bluebottle/surveys/urls/api.py

```diff
@@ -4,5 +4,5 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^(?P<slug>[\w-]+)/$', ProjectSurveyList.as_view(), name='project_survey_list'),
+    url(r'^$', ProjectSurveyList.as_view(), name='project_survey_list'),
```

Comment:
Is ember incompatible with this way of specifying the url? 

Required query arguments feel a little wrong to me....